hackathon_site/dashboard/frontend/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -75,7 +99,7 @@ const Dashboard = () => {
                             key={0}
                             data-testid="team"
                         >
-                            <TeamCard handleEditTeam={() => alert("Editing Team")} />
+                            <TeamCard handleEditTeam={() => openEditTeamModal()} />
```

Comment:
Because this function has no arguments you don't need to wrap a function around it 

`handleEditTeam={openEditTeamModal}` is sufficient